generate_scripts.py

```diff
@@ -30,26 +28,37 @@ def render(file_prefix, template_file, browser_executable):
                 navigation_cycles=30, 
                 per_navigation_delay=15, 
                 delay=3600, 
-                browser=browser_executable))
+                browser=process_name))
 
-for template_file in ['open_background', 'idle_on_site', 'idle', 'scroll', 'navigation', 'alligned_timers', 'zero_window']:
 
-    with open("./driver_scripts_templates/"+template_file) as file_:
-        template = Template(file_.read())
-        
-        # Generate for all Chromium based browsers
-        for browser in ['Chrome', 'Canary', "Chromium", "Edge"]:
-            browser_executable = utils.browsers_definition[browser]["executable"]
-            render(browser, template_file, browser_executable)
+def render_runner_scripts():
+    for template_file in ['open_background', 'idle_on_site', 'idle', 'scroll', 'navigation', 'alligned_timers', 'zero_window']:
 
-    # Skip alligned timer case as chrome only
-    if template_file == "alligned_timers":
-        continue
+        with open("./driver_scripts_templates/"+template_file) as file_:
+            template = Template(file_.read())
+            
+            # Generate for all Chromium based browsers
+            for browser in ['Chrome', 'Canary', "Chromium", "Edge"]:
+                process_name = utils.browsers_definition[browser]["process_name"]
+                render(browser, template, template_file, process_name)
 
-    template_file = "safari_"+template_file
-    with open("./driver_scripts_templates/"+template_file) as file_:
-        template = Template(file_.read())
+        # Skip alligned timer case as chrome only
+        if template_file == "alligned_timers":
+            continue
 
-        # Generate for Safari
-        render("", template_file, "")
+        template_file = "safari_"+template_file
```

Comment:
It says chrome only above, but safari here?

Reply:
The chrome comment is about generating only for chrome. Since the rest of the functions concerns safari if you `continue` you don't do if for safari.

Reply:
Ack